<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_answer</class>
 <widget class="QDialog" name="d_answer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d_answer {
    border-image: url(:/bg/resource/bg_most_1.png);
}

#label,#label_ask_user{
    color: white;
}
#label_Qtime,#label_ask_QueTitle{
    color: #F5EEEE;
}
#pushButton_ask_return{
    border-image: url(:/icons/resource/back.png);
}

#pushButton_ask_return:hover{
    border-image: url(:/icons/resource/back_h.png);
}
#textBrowser_ask_Qtext{
    border: none;
    background-color: rgb(146,168,209);
}
#textBrowser_ask_Atext{
    border: none;
    background: #F5F1F1
}
#textBrowser_ask_Qtext{
	color: #F5EEEE;
}</string>
  </property>
  <widget class="QLabel" name="label_ask_user">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>13</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ask_Atext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_ask_praiseNum">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>676</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ask_praise">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>666</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ask_return">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ask_focus">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ask_QueTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ask_Qtext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>3</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回答者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>676</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>赞数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Qtime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
